src/olympia/reviewers/tests/test_models.py

```diff
@@ -1492,3 +1493,51 @@ def test_verdict_info_prettifier(self):
 
         result = list(AutoApprovalSummary.verdict_info_prettifier({}))
         assert result == []
+
+
+class TestWhiteboardWatchChange(TestCase):
+
+    def make_addon(self, whiteboard='', **kwargs):
+        addon = Addon(type=amo.ADDON_EXTENSION, status=amo.STATUS_PUBLIC,
+                      **kwargs)
+        addon.save()
+        addon.versions.create(has_info_request=True)
+        addon.versions.create(has_info_request=False)
+        addon.versions.create(has_info_request=True)
+
+        whiteboard = Whiteboard(pk=addon.pk, public=whiteboard)
+        whiteboard.save()
+
+        return addon
+
+    def assertHasInfoSet(self, addon):
+        assert any([v.has_info_request for v in addon.versions.all()])
+
+    def assertHasInfoNotSet(self, addon):
+        assert all([not v.has_info_request for v in addon.versions.all()])
```

Comment:
We try to not follow unittest2 `assertCamelCase` style most of the times, but name these `assert_has_info_not_set` imho. Unless it's a generic helper in `amo.tests:TestCase`

Reply:
This was copy/paste. I'll take care though.